lib/esp32.toit

```diff
@@ -100,3 +100,19 @@ image_config_ -> ByteArray:
 
 set_real_time_clock_ seconds/int ns/int -> none:
   #primitive.core.set_real_time_clock
+
+/**
+Sets the RTC user data to the given $data starting from the given $offset.
+
+The given $data must be within bounds. That is 0 <= $offset <= $offset + data.size < 2048.
```

Comment:
I would add a constant `RTC_SIZE` and then reference it here:
```suggestion
The given $data must be within bounds. That is 0 <= $offset <= $offset + data.size < $RTC_SIZE.
```

Reply:
Called it `RTC_MEMORY_SIZE`.